### PR TITLE
Update to 1.5.0.pre (required because of ABI change)

### PR DESCRIPTION
**Pull request by kszucs**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy`
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
-{% set version = "1.4.0" %}
-{% set commit = "b62c22e52d37fa9580d85c5a2a2d73bd8e268364" %}
-{% set sha256sum = "2fa5704fc1164f68508e3050d30a6436dd56434e1a592d8ddf4afd68f62a7c01" %}
+{% set version = "1.4.0.post0" %}
```
Comment (kszucs) [inline recipe/meta.yaml]: @wesm `1.4.1`, `1.5.0` or `1.4.0.post0`?

**Comment by wesm** [comment]
There's not much point to opening a PR until the arrow-cpp build is done and merged, as you'll just be clogging up conda-forge with failing builds

File: recipe/meta.yaml (hunk 1 above)
Comment (wesm) [reply recipe/meta.yaml]: I recommend 1.5.0pre to avoid conflicting with 0.9.0 installs

Comment (cpcloud) [reply recipe/meta.yaml]: `post`, since there has not been a parquet-cpp release since `1.4.0`.

**Comment by kszucs** [comment]
It's easier to highlight questionable parts, like the version number - even if the build itself is failing.

File: recipe/meta.yaml (hunk 1 above)
Comment (kszucs) [reply recipe/meta.yaml]: > Wes has responded later (7s) but Phillip's comment is at the bottom (might be a github bug?)

I guess this should be `1.5.0pre` then.

Comment (cpcloud) [reply recipe/meta.yaml]: LGTM

**Comment by kszucs** [comment]
depends on https://github.com/conda-forge/arrow-cpp-feedstock/pull/56

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
-{% set version = "1.4.0" %}
-{% set commit = "b62c22e52d37fa9580d85c5a2a2d73bd8e268364" %}
-{% set sha256sum = "2fa5704fc1164f68508e3050d30a6436dd56434e1a592d8ddf4afd68f62a7c01" %}
+{% set version = "1.5.0.pre" %}
+{% set commit = "63f41b00bddecb172bd5b3aa0366b4653f498811" %}
```
Comment (kszucs) [inline recipe/meta.yaml:2]: Pinned the current [HEAD](https://github.com/apache/parquet-cpp/commit/63f41b00bddecb172bd5b3aa0366b4653f498811)

**Comment by kszucs** [comment]
Appveyor hasn't picked it up and `arrow-cpp` is still [queued](https://ci.appveyor.com/project/conda-forge/arrow-cpp-feedstock/branch/master)

**Comment by cpcloud** [comment]
LGTM +1. Merging.

**Pull request merged**